libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/SlackAdapter.cs

```diff
@@ -345,6 +346,10 @@ public async Task ProcessAsync(HttpRequest request, HttpResponse response, IBot
                         }
                         else
                         {
+                            // Convert Slack ts format to DateTime
```

Comment:
what's a `ts`? it would be good if the comment was more clear

Reply:
Is used across the documentation to signify "timestamp", but I'll change it to make it clearer.